<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Richard Collins &lt;richard.collins@bristol.ac.uk&gt;</author>
 <comment>Copyright (C) University of Bristol 2017
   This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. 
   If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
   See LICENSE file for details.</comment>
 <class>HSMPinDialog</class>
 <widget class="QDialog" name="HSMPinDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login to HSM Token</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="4" column="1">
    <widget class="QLineEdit" name="password">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QRadioButton" name="userTypeUser">
     <property name="text">
      <string>&amp;User</string>
     </property>
     <property name="icon">
      <iconset theme="user">
       <normaloff>.</normaloff>.</iconset>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">userType</string>
     </attribute>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QRadioButton" name="userTypeSO">
     <property name="text">
      <string>Securit&amp;y Officer</string>
     </property>
     <property name="icon">
      <iconset theme="dialog-warning">
       <normaloff>.</normaloff>.</iconset>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">userType</string>
     </attribute>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QRadioButton" name="userTypeCS">
     <property name="text">
      <string>Conte&amp;xt Specific</string>
     </property>
     <property name="icon">
      <iconset theme="dialog-question">
       <normaloff>.</normaloff>.</iconset>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">userType</string>
     </attribute>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>User</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="tokenLabel">
     <property name="text">
      <string>Token</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Log into</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>HSMPinDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>HSMPinDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="userType"/>
 </buttongroups>
</ui>
